streets_reader/src/lib.rs

```diff
@@ -52,16 +54,21 @@ pub fn detect_country_code(streets: &mut StreetNetwork) {
         return;
     }
 
-    let geocoder = country_geocoder::CountryGeocoder::new();
-    let pt = streets.gps_bounds.get_rectangle()[0].into();
+    let geocoder = CountryBoundaries::from_reader(BOUNDARIES_ODBL_60X30).unwrap();
```

Comment:
I opted to use the smallest, but slowest raster here, which is 296 KB. We could opt for a larger, but faster raster too.